lib/pf/services/manager/haproxy.pm

```diff
@@ -196,11 +242,6 @@ sub stop {
     return $result;
 }
 
-sub isManaged {
-    my ($self) = @_;
-    return $cluster_enabled;
```

Comment:
Yes true, but now since we manage mariadb can we configure it to listen on the mgmt ip and keep haproxy listen on 127.0.0.1:3306 ?